src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -130,40 +131,37 @@ const BioAndChat = (props) => {
           }
         />
       </TouchableOpacity>
-      <TouchableOpacity onPress={toggleSwitch} style={styles.toggleSwitchContainer}>
-        <ToggleSwitch
-          value={isAnonimity}
-          onValueChange={toggleSwitch}
-          labelLeft={
-            isAnonimityEnabled || !isSignedMessageEnabled ? 'Anonymity' : 'Anonymity disabled'
-          }
-          styleLabelLeft={{color: isAnonimityEnabled ? colors.white : '#648ABF'}}
-        />
-      </TouchableOpacity>
+      <ToggleSwitchAnon
+        value={isAnonimity}
+        onValueChange={toggleSwitch}
+        labelLeft={
+          isAnonimityEnabled || !isSignedMessageEnabled ? 'Anonymity' : 'Anonymity disabled'
+        }
+      />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ToggleSwitchAnon` component is being used here instead of the previous `ToggleSwitch` component. Ensure that this new component has the same props and behaves as expected in the context of this code. 



<!-- This is an auto-generated comment by OSS CodeRabbit -->